client/js/components/display.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (state, prev, send) {
   send('preview:update', {isLoading: true})
 
   if (entryName && archive) {
-    send('preview:update', {error: {message: 'Looking for peers...', body: 'If it is taking a long time, use the desktop app.'}})
+    send('preview:update', {error: {message: 'Looking for peers …', body: 'If this is taking a long time, use the desktop app.'}})
```

Comment:
This is where right now it seems like no webrtc option will be supported so we will have a different message, like "Use the desktop app to download files"